### PR TITLE
Update ORT to 1.5.1 in Bert Quantization Notebook

### PR DESCRIPTION
**Pull request by yufenglee**
**Description**: Describe your changes.
Update ORT to 1.5.1 in Bert Quantization Notebook

### CODE REVIEW

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 1]
```diff
@@ -929,21 +944,14 @@
...
-    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. Comparing with full precision, PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.34x speedup. And ORT quantization can achieve ~1.27x speedup, comparing with PyTorch quantization. \n",
+    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. Comparing with PyTorch full precision, PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.73x speedup. And ORT quantization can achieve ~1.33x speedup, comparing with PyTorch quantization. \n",
     "You can run the [benchmark.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/benchmark.py) for comparison on more models.\n",
     "\n",
     "|Engine | Full Precision Latency(s) | Quantized(s) |\n",
     "| --- | --- | --- |\n",
-    "| PyTorch 1.6 | 32.8 | 25.5 |\n",
-    "| ORT 1.4 | 25.6 | 19.5 |"
+    "| PyTorch 1.6 | 33.8 | 22.5 |\n",
+    "| ORT 1.4 | 26.0 | 19.5 |"
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:953]: ORT 1.5?

**Review by tianleiwu — APPROVED** [review]

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 2]
```diff
@@ -916,34 +931,27 @@
...
-    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. Comparing with full precision, PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.34x speedup. And ORT quantization can achieve ~1.27x speedup, comparing with PyTorch quantization. \n",
+    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. Comparing with PyTorch full precision, PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.73x speedup. And ORT quantization can achieve ~1.33x speedup, comparing with PyTorch quantization. \n",
     "You can run the [benchmark.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/benchmark.py) for comparison on more models.\n",
     "\n",
     "|Engine | Full Precision Latency(s) | Quantized(s) |\n",
     "| --- | --- | --- |\n",
-    "| PyTorch 1.6 | 32.8 | 25.5 |\n",
-    "| ORT 1.4 | 25.6 | 19.5 |"
+    "| PyTorch 1.6 | 33.8 | 22.5 |\n",
+    "| ORT 1.5 | 26.0 | 19.5 |"
```
Comment (thiagocrepaldi) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:953]:
```suggestion
    "| ORT 1.5.1 | 26.0 | 19.5 |"
```

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 3]
```diff
@@ -916,34 +931,27 @@
     "| Engine | Full Precision(MB) | Quantized(MB) |\n",
     "| --- | --- | --- |\n",
     "| PyTorch 1.6 | 417.7 | 173.1 |\n",
-    "| ORT 1.4 | 417.7 | 106.5 |\n",
+    "| ORT 1.5 | 417.7 | 106.5 |\n",
```
Comment (thiagocrepaldi) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:934]:
```suggestion
    "| ORT 1.5.1 | 417.7 | 106.5 |\n",
```

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 4]
```diff
@@ -916,34 +931,27 @@
...
     "\n",
     "### Accuracy\n",
     "OnnxRuntime achieves a little bit better accuracy and F1 score, even though it has small model size.\n",
     "\n",
-    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.4 Quantization |\n",
+    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.5 Quantization |\n",
```
Comment (thiagocrepaldi) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:939]:
```suggestion
    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.5.1 Quantization |\n",
```

Comment (yufenglee) [reply onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:939]: Usually, patch version is not specified in benchmark

**Pull request merged**